wpilibc/src/main/native/cpp/DoubleSolenoid.cpp

```diff
@@ -165,8 +166,9 @@ DoubleSolenoid::Value DoubleSolenoid::Get() const {
  */
 bool DoubleSolenoid::IsFwdSolenoidBlackListed() const {
   int blackList = GetPCMSolenoidBlackList(m_moduleNumber);
-  return (blackList & m_forwardMask) ? 1 : 0;
+  return blackList & m_forwardMask;
```

Comment:
I'm not a fan of the implicit cast in this scenario.  What about `(blackList & m_forwardMask) != 0`?